REPLICATION_POLICIES.md

```diff
@@ -0,0 +1,293 @@
+
+###ReplicationService
+`ReplicationService` is an abstract class and your subclass must implement the following abstract method:
+* `protected abstract Replicator[] getReplicators(Context context);` This method should return an array of `Replicator` objects that define the replications you want to invoke.
+
+###PeriodicReplicationService
+This abstract class is a child of `ReplicationService` and requires you to implement the following abstract methods:
+* `protected abstract Replicator[] getReplicators(Context context);` This method should return an array of `Replicator` objects that define the replications you want to invoke.
+* `protected abstract int getBoundIntervalInSeconds();` This method should return the interval (in seconds) you wish to have between replications when components are bound to the Service.
+* `protected abstract int getUnboundIntervalInSeconds();` This method should return the interval (in seconds) you wish to have between replications when components are not bound to the Service.
+* `protected abstract boolean startReplicationOnBind();` This should return `true` if you wish to have replications triggered immediately when a component binds to the Service.
+
+Note that internally this uses [android.app.AlarmManager.setInexactRepeating()](http://developer.android.com/reference/android/app/AlarmManager.html#setInexactRepeating(int, long, long, android.app.PendingIntent)) 
+to schedule the repetition of the replications at the given intervals in a battery efficient way, this means the intervals between replications will not be exact.
+
+###PeriodicReplicationReceiver
+This uses a `WakefulBroadcastReceiver` to trigger periodic replications at the intervals specified by your
+`PeriodicReplicationService`. This means your application does not have to keep running to trigger replications at the
+intervals you require, but will be restarted at the time a replication is required. This class also handles resetting of
+the periodic replications after a reboot of the device. You must add the following `intent-filter`s to your
+`PeriodicReplicationReceiver`'s entry in the `AndroidManifest.xml` to enable periodic replication:
+
+```xml
+<action android:name="com.cloudant.sync.replication.PeriodicReplicationReceiver.Alarm" />
+<action android:name="android.intent.action.BOOT_COMPLETED" />
+```
+
+##Controlling the replication service
+To control the replication service you start the service passing an Extra in the `Intent` used to start the service whose
+key is `ReplicationService.EXTRA_COMMAND`, and whose value is one of:
+
+* `ReplicationService.COMMAND_START_REPLICATION` This is used to start the replicators.
+* `ReplicationService.COMMAND_STOP_REPLICATION` This is used to stop replicators in progress.
+* `PeriodicReplicationService.COMMAND_START_PERIODIC_REPLICATION` This only applies if you're using the `PeriodicReplicationService`. This starts the periodic replications.
```

Comment:
You could maybe shorten these to `This stops the periodic replications when you are using the `PeriodicReplicationService`.